app/services/schema_data.py

```diff
@@ -0,0 +1,84 @@
+from biocypher import BioCypher
+import logging
+import yaml
+
+# Setup basic logging
+logging.basicConfig(level=logging.DEBUG)
+
+class SchemaManager:
+    def __init__(self, schema_config_path: str, biocypher_config_path: str):
+        self.bcy = BioCypher(schema_config_path=schema_config_path, biocypher_config_path=biocypher_config_path)
+        self.schema = self.bcy._get_ontology_mapping()._extend_schema()
+        self.parent_nodes = ["position entity", "coding element", "non coding element", "genomic variant", "epigenomic feature", "3d genome structure", "ontology term", "chromosome chain"]
```

Comment:
consider automating the parent nodes and edges list generation from the schema. Maybe in another PR